applications/vila_live/metadata.json

```
@@ -0,0 +1,36 @@
+{
+	"application": {
+		"name": "VILA Live",
+		"authors": [
+			{
+				"name": "Holoscan Team",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"language": "Python",
+		"version": "1.0.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "1.0.0",
+			"tested_versions": [
+				"1.0.0"
+			]
+		},
```

Was this tested with Holoscan SDK 2.0.0?

```suggestion
		"holoscan_sdk": {
			"minimum_required_version": "2.0.0",
			"tested_versions": [
				"2.0.0"
			]
		},
```

---

No it was not

---

It looks like this is using the default base image as set by `dev_container`, which is for Holoscan SDK v2.0.0.

Could you please either:
- Specify the Holoscan SDK version used (`1.0.3` is public, `1.0.0` is not) and include README instructions on how to build with that base image; or
- Test and verify with Holoscan SDK v2.0.0 as the default?